nuxt.config.ts

```diff
@@ -44,10 +44,6 @@ export default {
    ** Customize the progress-bar color
    */
   loading: { color: '#002E5D' },
-  /*
-   ** Global CSS
-   */
-  css: ['@/assets/style/app.sass'],
```

Comment:
Where did this go?

Reply:
You took it out a few commits ago: e8b0252
This has been broken since then.